### PR TITLE
refactor: add more explicit types to KineticSdk

### PR DESCRIPTION
**Pull request by beeman**

### CODE REVIEW

**Review by RamirezAlex — APPROVED** [review]

**Comment by RamirezAlex** [comment]
We still have a few typing issues. Just 3 :)

File: libs/sdk/src/lib/kinetic-sdk.ts [hunk 1]
```diff
@@ -24,47 +25,47 @@ export class KineticSdk {
     this.internal = new KineticSdkInternal(sdkConfig)
   }
 
-  get endpoint() {
+  get config(): AppConfig {
```
Comment (RamirezAlex) [inline libs/sdk/src/lib/kinetic-sdk.ts:28]: This `get config(): AppConfig | undefined` make the typing issue go away.

File: libs/sdk/src/lib/kinetic-sdk.ts [hunk 2]
```diff
@@ -24,47 +25,47 @@ export class KineticSdk {
...
-  config() {
-    return this.internal.appConfig
-  }
-
   createAccount(options: CreateAccountOptions): Promise<AppTransaction> {
     return this.internal.createAccount(options)
   }
 
-  getExplorerUrl(path: string) {
+  getExplorerUrl(path: string): string {
```
Comment (RamirezAlex) [inline libs/sdk/src/lib/kinetic-sdk.ts:52]: `getExplorerUrl(path: string): string | undefined`

File: libs/sdk/src/lib/kinetic-sdk.ts [hunk 3]
```diff
@@ -24,47 +25,47 @@ export class KineticSdk {
...
+    return this.internal.appConfig
+  }
+
+  get endpoint(): string {
     return this.sdkConfig.endpoint
   }
 
-  get solanaRpcEndpoint() {
-    return this.sdkConfig.solanaRpcEndpoint || 'mainnet-beta'
+  get solanaRpcEndpoint(): string {
```
Comment (RamirezAlex) [inline libs/sdk/src/lib/kinetic-sdk.ts:36]: `get solanaRpcEndpoint(): string | undefined`

**Comment by RamirezAlex** [comment]
🎉 

Feel free to merge

**Comment by beeman** [comment]
> 🎉
> 
> Feel free to merge

Thanks Alex! 

This PR introduced a warning for exporting the types, I created a follow-up issue here #302 

image

**Pull request merged**